src/types/Firestore.ts

```diff
@@ -80,17 +80,18 @@ export interface FirestoreOrdered {
   eventPurchase: Array<WithId<Purchase>>;
   events?: Array<WithId<VenueEvent>>;
   experiences: Array<WithId<Experience>>;
-  parentVenueEvents: Array<WithId<VenueEvent>>;
+  parentVenue: Array<WithId<AnyVenue>>;
```

Comment:
Since `parentVenue` seems new, it should be safe to fix the type on this:

```suggestion
  parentVenue?: Array<WithId<AnyVenue>>;
```